pages/en/lb4/Decorator.md

```diff
@@ -0,0 +1,209 @@
+---
+lang: en
+title: 'Decorators'
+keywords: LoopBack 4.0, LoopBack-Next
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Decorators.html
+summary:
+---
+
+A decorator allows you annotate or modify your class declarations and members with metadata.
+
+## Introduction
+
+*If you're new to Decorators in TypeScript, see [here](https://www.typescriptlang.org/docs/handbook/decorators.html) for more info.*
+
+Decorators give LoopBack the flexibility to modify your plain TypeScript classes 
+and properties in a way that allows the framework to better understand how to 
+make use of them, without you needing to inherit base classes or add functions 
+that tie into an API.
+
+Currently, LoopBack core provides built-in decorator functions you can use directly
+by `@myDecorator`, e.g. Route decorators that introduces in section Route-Decorators, 
+and implements a dependency injection system that can apply a decorator by `@inject('myDecorator')`. 
+
+As a default, LoopBack comes with some pre-defined component decorators for your use:
+- [authentication](#authentication-decorators)
+- [repository](#repository-decorators)
+
+## Route Decorators
+- @get, @post, @put, @patch, @del
+
+  Register route for a controller by rest decorator, for their usage, refer to 
+  [Routing to Controllers](controllers.htm#Routing-to-Controllers)
+
+- @api
+
+  Register route for a controller by api decorator, more details and example refer 
+  to [Specifying Controller APIs](controllers.htm#Specifying-Controller-APIs)
+
+- @operation
+
+  Expose a Controller method as a REST API operation. Usage refer to [TBD in Controllers](controllers.htm#)
+
+- @param
+
+  Describe an input parameter of a Controller method. 
+  
+  `@param` can be applied to method itself or specific parameters. For example,
+
+  ```ts
+    class MyController {
+      @get('/')
+      @param(offsetSpec)
+      @param(pageSizeSpec)
+      list(offset?: number, pageSize?: number) {}
+    }
+  ```
+
+  or
+  
+  ```ts
+    class MyController {
+      @get('/')
+      list(
+        @param(offsetSpec) offset?: number,
+        @param(pageSizeSpec) pageSize?: number,
+      ) {}
+    }
+  ```
+  
+  More example can be found in [Writing Controller methods](controller.htm#Writing-Controller-methods)
+  
+## Dependency Injection
+
+@inject is a decorator to annotate method arguments for automatic injection by 
+LoopBack IoC container. @inject can only be used on properties or method parameters.
```

Comment:
I think you should to wrap `@inject` in back-ticks, so that it renders as code. I think GitHub's syntax highlighter thinks `@inject` is a GitHub handle now.